client/components/tinymce/plugins/wpcom-autoresize/plugin.js

```diff
@@ -24,6 +24,27 @@ function wcpomAutoResize( editor ) {
 		return;
 	}
 
+	function isEndOfEditor() {
+		var range, start, body, element, child;
+		range = editor.selection.getRng();
+
+		if ( ( range.startOffset === 0 && range.endOffset !== 0 ) || ! range.collapsed ) {
+			return false;
+		}
+
+		start = range.startContainer;
+		body = editor.getBody();
+		element = start;
+		do {
+			child = element;
+			element = element.parentNode;
+			if ( element.childNodes[ element.childNodes.length - 1 ] !== child ) {
```

Comment:
I see this function checks that we're focusing the last dom node of the editor, but I couldn't understand if we're checking that we're at the last character of this dom node. Shouldn't we compare `range.endOffset` with the content of the `range.endContainer`?